src/lib/product_reviews/visualizer/index.js

```diff
@@ -60,50 +60,16 @@ const ReviewCount = styled.h3`
 class Visualizer extends Component {
    render() {
       const { productReviews, reviewsUrl } = this.props;
-      const { average, groupedReviews } = productReviews;
+      const { average, groupedReviews, jsonld } = productReviews;
       const numReviews = productReviews.count;
 
-      const schemaReviews = productReviews.reviews
-         .slice(0,10)
-         .map(r => {
-            return {
-               "@type": "Review",
-               "datePublished": r.date,
-               "reviewRating": {
-                  "@type": "Rating",
-                  "ratingValue": r.rating,
-                  "bestRating": "5"
-               },
-               "author": {
-                  "@type": "Person",
-                  "name": r.author.name
-               }};
-       });
-
-      const reviewJsonld = {
-         "@context": "http://schema.org/",
-         "@type": "Product",
-         "url": window.location.href,
-         "aggregateRating": {
-           "@type": "AggregateRating",
-           "ratingValue": average,
-           "reviewCount": numReviews
-         },
-         "review": [schemaReviews],
-      };
-
-      if (numReviews) {
-         // This is the only way to get the product name with current data provided
-         reviewJsonld["name"] = productReviews.reviews[0].productName;
-      }
-
       return (
          <Container>
-            <Helmet>
-               {numReviews > 0 &&
-                  <script type="application/ld+json">{JSON.stringify(reviewJsonld)}</script>
-               }
-            </Helmet>
+            {Object.keys(jsonld).length && (
+               <Helmet>
+                  <script type="application/ld+json">{JSON.stringify(jsonld)}</script>
+               </Helmet>
+            )}
```

Comment:
Doesn't React output 0 here if length is 0?

Reply:
You're totally right, thanks for catching that

Reply:
https://github.com/iFixit/product-page-components/pull/41/commits/b7454f3f0f26b25b06aa0f7bb14828dd75de189f